src/construction/Slider/Slider.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import Flickity from 'react-flickity-component';
+import logoSlider from '../../image/LogoSlider.png'
+import './slider.scss'
+import girlOne from '../../image/girls-1.jpg'
+import girlsTwo from '../../image/girls-2.png'
+import girlsThree from '../../image/girls-3.png'
+import iconsArrow from '../../image/iconSlider.png'
+import flickity from '../../modules/flickity.module.scss'
+
+const Slider = () => {
+	const flickityOptions = {
+		initialIndex: 2,
+	};
+
+	const girls = {
+		id: [1, 2, 3],
+		image: [girlOne, girlsTwo, girlsThree]
+	}
+	return (
+		<div className={flickity.sliderBody}>
+			<div className={flickity.container}>
+				<div className={flickity.SliderTextBg}><img src={logoSlider} alt="" /></div>
+				<Flickity
+					className={'Slider'}
+					elementType={'div'}
+					disableImageLoaded={false}
+					options={flickityOptions}
+					reloadOnUpdate
+					static
+				>
+					{girls['id'].map((i) => {
+						return (
```

Comment:
You don't need to access the id and map it (what if the id will not be numbers) 

You can do something like this : 

```js
girls.map((item,index) => ...))
```